VK_V1.0/VK_V1.0/View/NewsLikeControl.swift

```diff
@@ -0,0 +1,69 @@
+// NewsLikeControl.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Компонент для отображения количества лайков в ячейке Новости
+final class NewsLikeControl: UIControl {
```

Comment:
```suggestion
final class LikeControl: UIControl {
```
Он разве не универсальный зачем только для News?

Reply:
нет, не универсальный, разное кол-во тапов, анимация и передаваемые значения